src/detection.c

```diff
@@ -43,8 +79,10 @@ static void sm_emitter_adc(void)
 		sensors_off[SENSOR_SIDE_RIGHT] = adc_read_injected(ADC1, 2);
 		sensors_off[SENSOR_FRONT_LEFT] = adc_read_injected(ADC1, 3);
 		sensors_off[SENSOR_FRONT_RIGHT] = adc_read_injected(ADC1, 4);
-		gpio_toggle(GPIOA, GPIO8 | GPIO9);
-		gpio_toggle(GPIOB, GPIO8 | GPIO9);
+		side_left_toggle();
+		side_right_toggle();
+		front_left_toggle();
+		front_right_toggle();
```

Comment:
I'd rather use on/off, instead of toggle.

Reply:
Clear and Set instead of Toogle and macros instead of functions.